### PR TITLE
[swagger] - add operation `swagger_context` example.

### PR DESCRIPTION
**Pull request by gorghoa**
Hi :)

This PR adds an example of `swagger_context` customization for operations. While a reader may extrapolate from the `properties` example, I admit I had not! Hence this PR.

I’ve added a screenshot since  I think it will help the newcomer reader to build a mental mapping between the swagger definitions and the impact on UI. 

Also demonstrate markdown usage in `description`.

### CODE REVIEW

File: core/swagger.md [hunk 1]
```diff
@@ -225,6 +225,34 @@ class User
...
+            ![A great rabbit](https://rabbits.com/dark.jpeg)
+
+          requestBody: '{"days": 23}'
+          parameters:
+            - {name: 'theme', description: 'dark'}
+```
+
+![Impact on swagger ui](../distribution/images/swagger-ui-2.png)
+
+
```
Comment (Simperfit) [inline core/swagger.md]: to remove

File: core/swagger.md [hunk 2]
```diff
@@ -225,6 +225,34 @@ class User
...
+
```
Comment (Simperfit) [inline core/swagger.md]: to remove

**Comment by gorghoa** [comment]
thanks for your review @Simperfit :)

File: core/swagger.md [hunk 3]
```diff
@@ -225,6 +225,32 @@ class User
...
+      create_user:
+        method: get
+        path: '/rabbit/rand'
+        controller: App\Controller\RandomRabbit
+        swagger_context:
+          summary: Random rabbit picture
+          description: >
+            # Pop a great rabbit picture by color!
+
+            ![A great rabbit](https://rabbits.com/dark.jpeg)
```
[the remark this answers is not in the document]
Comment (gorghoa) [reply core/swagger.md]: done @meyerbaptiste !

https://rabbit.org/graphics/fun/netbunnies/jellybean1-brennan1.jpg

**Comment by dunglas** [comment]
Thanks @gorghoa

**Pull request merged**